content/03.methods.md

```diff
@@ -43,10 +43,13 @@ Following training, we generated a document vector for every article within bioR
 This document vector is calculated by taking the average of every token present within a given article, ignoring those that were absent from the word2vec model.
 
 #### Dimensionality Reduction of Document Embeddings
-1. Explain how tSNE works (paragraph one)
-2. Explain how PCA works  (paragraph two)
-3. Discuss how words were mapped onto PC components via cosine similarity
-4. ^ Explain cosine similarity
+We used t-Distributed Stochastic Neighbor Embedding (tSNE) [@raw:tsne_Maaten] to generate a global view of the bioRxiv preprint landscape. 
+This model was trained on bioRxiv document vectors using scikit-learn's [@raw:scikit-learn] implementation with a random seed of 100 and default settings for all other parameters.
+
+We used principal component analysis (PCA) [@doi:10.1111/1467-9868.00196] to project bioRxiv document vectors into a low dimensional space.
+This model was trained using scikit-learn's implementation with a random seed of 100, output of 50 principal components, and default settings for all other parameters.
```

Comment:
Is this using a randomized solver?

Reply:
Yup. The default is for PCA to figure out which algorithm is best. After reading the documentation, I found that scikit-learn's solver determined that randomized solver is most optimal.